src/components/InputTag/InputTag.module.scss

```diff
@@ -46,7 +46,7 @@
 .inputTagWrapperOn {
```

Comment:
Put common values of inputTagWrapperOn and inputTagWrapperOff in a class named inputTagWrapper and the make separator .on and .off classes